.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: pip
```

Comment:
I read that they support poetry, but I guess there is no option for that so this is correct?

Reply:
the dashboard automatically checked the pyproject.toml. I assumed that v2 will do so as well